src/styles/header.scss

```diff
@@ -0,0 +1,91 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+$header-height: 60px;
+$header-padding: 0 50px;
+$logo-height: 40px;
+$logo-size: 40px;
+$logo-margin: 10px;
+$link-color: #060b35;
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
+  background: #e5e5e5;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: $header-height;
+  background: #fff;
+  padding: $header-padding;
+  
+  &__logo {
+    height: $header-height;
+    
+    &__pic {
+      display: inline-block;
+      background-image: url(images/logo.png);
+      background-repeat: no-repeat;
+      height: $logo-height;
+      width: $logo-size;
+      margin-top: $logo-margin;
+    }
+  }
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  height: $header-height;
+  
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    text-transform: uppercase;
+  }
+
+  &__item {
+    padding-right: 20px;
+    
+    &.last {
+      padding-right: 0;
+    }
+  }
```

Comment:
2 points:
- paddings are not for this purpose, let's use margins
- use pseudo-classes